### DISCUSSION TITLE
OAuth error	"bad_verification_code" (Unusual)

### DISCUSSION THREAD

**Discussion by AbelFalcon** [community/community#123139] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

A few days ago, I created a system so that users could use GIthub as a login method. In principle I have not had any problem, obtaining the token of the users. And getting data from them. 

I have created an APi to manage all this from my client. But today, I keep getting "bad_verification_code" error and I don't understand why, even I tried to get my own token, with the verification code provided. 

```
app.get("/getToken", async (req, res) => {
  try {
    const { code } = req.query
    const params = `client_id=${CLIENT_ID_GITHUB}&client_secret=${CLIENT_SECRET_GITHUB}&code=${code}`
    const response = await fetch(
      "https://github.com/login/oauth/access_token",
      {
        method: "POST",
        headers: {
          "Content-Type": "application/x-www-form-urlencoded",
          Accept: "application/json",
        },
        body: params,
      }
    )

    const data = await response.json()
    res.json(data)
  } catch (error) {
    console.error("Error al obtener el token:", error)
    res.status(500).json({ error: "Hubo un error al obtener el token" })
  }
})
```
I have checked the parameters. And the code coming from my client, is passed correctly. 

![imagen](https://github.com/community/community/assets/71991536/fbe9dd75-057e-4766-997b-ef0d692b162f)

**Comment by MCWXT** [comment]
I have also encountered this problem. How did you solve it?

**Reply by AbelFalcon** [reply, marked as answer]
I'm sure you've already figured out the problem. 

But just in case anyone runs into this in the future. I was having rste limit problems. One day I made so many attempts (having a portion of the code wrong) that it ends up going over the limit.
